src/lib/network/lan_items_summary.rb

```diff
@@ -153,28 +154,20 @@ def static_summary
     #
     # @return [String] bridge configured interfaces summary
     def bridge_summary
-      _("Bridges: %s") % bridges.map { |n| "#{n} (#{LanItems.bridge_slaves(n).sort.join(", ")})" }
+      config = Y2Network::Config.find(:yast)
+      _("Bridges: %s") % config.interfaces.by_type("br").map do |n|
+        "#{n} (#{config.interfaces.bridge_slaves(n.name).sort.join(", ")})"
+      end
```

Comment:
are you sure it works when you have more bridges?

```
irb(main):003:0> "a %s" % [1, 2].map { |i| i }
=> "a 1"
```

And even if it works, it is a bit too much magic for me. So please use join on result of map and ideally prepare string in advance.

```
so something likes
```
bridges = config.interfaces.by_type("br").map do |n|
  "#{n} (#{config.interfaces.bridge_slaves(n.name).sort.join(", ")})"
end
_("Bridges: %s") % bridges.join(", ")
```